datahub-web-react/src/providers/EducationStepIdsAllowlistProvider.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { useState } from 'react';
+import { CURRENT_ONBOARDING_IDS } from '../app/onboarding/onboarding';
+import { EducationStepIdsAllowlistContext } from './EducationStepIdsAllowlistContext';
+
+export function EducationStepIdsAllowlistProvider({ children }: { children: React.ReactNode }) {
+    const [educationStepIdsAllowlist, setEducationStepIdsAllowlist] = useState<string[]>(CURRENT_ONBOARDING_IDS);
+
+    return (
+        <EducationStepIdsAllowlistContext.Provider value={{ educationStepIdsAllowlist, setEducationStepIdsAllowlist }}>
+            {children}
+        </EducationStepIdsAllowlistContext.Provider>
+    );
+}
```

Comment:
yeah I definitely think we should just toss this into the existing EducationStepsProvider and just have one Context and one Provider for Education Steps-related things

Reply:
Done!